### ISSUE TITLE
More tags for FlareSolverr image

### ISSUE THREAD

**Issue by GeoCookie** [FlareSolverr/FlareSolverr#171]
Hi there,

Currently, beside latest, we only got specific tags with major, minor and patch number.
It could be usefull having tags with only major/minor version and major version.

Some providers uses this model
graylog/graylog
mongo

Thanks for your work

**Comment by ngosang** [comment]
You are right but I'm not following versioning number as in other projects.
Each time Cloudflare changes the challenge or I fix a bug I release a patch/fix release and all users should update even if it's a fix version. That's why I don't have more tags.
To do the things right I should bump the minor number in each release. I'm open to suggestions to improve the release numbers first and the image tags later.

**Comment by GeoCookie** [comment]
Thank you for your reply.
Since your releases rarely contain only fixes, this is the minor number that should be bumped, I agree with you.
